### ISSUE TITLE
if there is no dicomWeb server

### ISSUE THREAD

**Issue by haoxl3** [knopkem/vtk-frontend#1]
Is there an open source DicomWeb Server? I'd like to see a demo?

**Comment by knopkem** [comment]
yes, dicomweb-proxy + regular PACs or dicomweb-pacs or dicomweb-archive. check my other repos
